langkit/templates/pkg_introspection_spec_ada.mako

```diff
@@ -295,6 +295,17 @@ package ${ada_lib_name}.Introspection is
    --  Return the lower-cased name for Property's argument whose index is
    --  Argument_Number.
 
+   function Evaluate_Property
```

Comment:
Same comment, although makes me think that `Eval_Property` is better than `Get_Property` which makes no sense